src/analysis/inner_product_space/adjoint.lean

```diff
@@ -434,14 +434,13 @@ open_locale complex_conjugate
 
 /-- The adjoint of the linear map associated to a matrix is the linear map associated to the
 conjugate transpose of that matrix. -/
-lemma conj_transpose_eq_adjoint (A : matrix m n 𝕜) :
-  to_lin' A.conj_transpose =
-  @linear_map.adjoint _ (euclidean_space 𝕜 n) (euclidean_space 𝕜 m) _ _ _ _ _ (to_lin' A) :=
+lemma to_euclidean_lin_conj_transpose_eq_adjoint (A : matrix m n 𝕜) :
+  A.conj_transpose.to_euclidean_lin = A.to_euclidean_lin.adjoint :=
 begin
```

Comment:
The statement of this lemma was the subject of some discussion in https://github.com/leanprover-community/mathlib/pull/11551#discussion_r787677895, and in [this Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/SVD/near/269383210).

Note the new version is now consistent with the statement of `matrix.is_hermitian_iff_is_symmetric` below.